focuspoints.lrdevplugin/DefaultDelegates.lua

```diff
@@ -55,19 +56,22 @@ end
 
 --[[
   -- method figures out the orientation the photo was shot at by looking at the metadata
-  -- returns 90, 270, or 0
+  -- returns 90, 270, or 0 (in trigonometric sense)
 --]]
 function DefaultDelegates.getShotOrientation(photo, metaData)
   local dimens = photo:getFormattedMetadata("dimensions")
   local orgPhotoW, orgPhotoH = parseDimens(dimens) -- original dimension before any cropping
 
   local metaOrientation = ExifUtils.findValue(metaData, "Orientation")
-  if (string.match(metaOrientation, "90") and orgPhotoW < orgPhotoH) then
+  if (string.match(metaOrientation, "90 CW") or string.match(metaOrientation, "90")) and orgPhotoW < orgPhotoH then
+    return 270
```

Comment:
why is this backwards? A metadata of "Rotate 90 CW" used to return a value of 90. and "Rotate 270 CW" used to return 270.

Also ```string.match(metaOrientation, "90 CW")``` is a duplicate of  ```string.match(metaOrientation, "90")``` "90" will match anything with a "90" in it.